bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/obs/SchedulingBlock.scala

```diff
@@ -30,9 +30,12 @@ object SchedulingBlock {
     apply(start, Some(duration))
 
   def apply(start: Long, duration: Option[Long]): SchedulingBlock =
-    new Impl(start, duration.filter(_ > 0))
+    new Impl(start, duration.filter(_ >= 0))
 
   def unsafeFromStrings(startString: String, durationString: String): SchedulingBlock =
     apply(startString.toLong, durationString.toLong)
 
+  def unsafeFromStrings(startString: String): SchedulingBlock =
```

Comment:
Yeah it's because they throw; a proper `.fromString` would return an `Option` which might be a better approach, but it's coming from PIO as Shane mentioned and would just end up throwing on `None` anyway.